idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/EditorConfig.kt

```diff
@@ -0,0 +1,23 @@
+package io.github.composegears.valkyrie.util.extension
+
+import com.intellij.openapi.project.Project
+import java.io.File
+
+internal fun Project.getEditorConfig(): File? = runCatching {
+    basePath?.let { path ->
+        File(path).resolve(".editorconfig").takeIf { it.exists() }
+    }
+}.getOrNull()
+
+internal fun File.getIndentSize(): Int? {
+    useLines {
+        it.forEach { field ->
+            if (field.startsWith("indent_size")) {
+                return field.replace(" ", "")
+                    .removePrefix("indent_size=")
+                    .toIntOrNull()
+            }
+        }
+    }
+    return null
+}
```

Comment:
I guess you can read properties by `java.util.Properties` instead of parsing manually.